login.html

```diff
@@ -0,0 +1,106 @@
+                            </a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="login.html" class="nav-link"><button class="btn btn-primary">Login</button></a>
+                        </li>
+                    </ul>
+                </div>
+            </nav>
+        </div>
+    </header>
+    <main>
+        <section class="my-5">
+            <div class="container">
+                <div class="row justify-content-center">
+                    <div class="col-lg-5 bg-white p-4 mb-5 rounded shadow-sm">
+                        <h1>Login</h1>
+                        <div class="d-flex">
+                            <p class="text-muted mr-1">Do not have an account?</p>
+                            <a href="#" class="login-link">Sign up</a>
+                        </div>
+                        <form action="profile.html">
+                            <div class="my-3">
+                                <label for="email" class="form-label">Email address</label>
+                                <input type="email" class="form-control" id="email" placeholder="name@example.com"
+                                    required />
+                            </div>
+                            <div class="mt-4">
+                                <label for="password" class="form-label">Password</label>
+                                <input type="password" class="form-control" id="password"
+                                    placeholder="Enter your password" required />
+                            </div>
+                            <div class="d-flex justify-content-between mt-3">
+                                <div>
+                                    <input type="checkbox" name="" id="rememberMe" />
+                                    <label for="rememberMe">Remember me</label>
+                                </div>
+                                <a href="#" class="login-link">Forgot your password?</a>
+                            </div>
+                            <button type="submit" class="btn btn-primary mt-5">
+                                Login
+                            </button>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        </section>
+    </main>
+    <footer>
+        <div class="text-center bg-white p-4">© 2022 Copyright</div>
+    </footer>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
+        integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.12.9/dist/umd/popper.min.js"
+        integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q"
+        crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/js/bootstrap.min.js"
```

Comment:
```suggestion
    <script src="/node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
```

This includes popper and jQuery